lib/docx/body-reader.js

```diff
@@ -491,3 +505,11 @@ function joinElements(first, second) {
 function identity(value) {
     return value;
 }
+
+function clearState() {
+    lastInstrText = null;
+}
+
+function parseHyperlink(instrText) {
```

Comment:
Being a bit picky, but `parseHyperlink` feels a bit vague. Specifically, this is actually parsing the `w:instrText` text, whatever the correct term for that is. I think it might be field codes, but the spec should say.

In other words, perhaps something like: `parseHyperlinkFieldCode` (if they are indeed called field codes).